src/nunjucks/moduk/components/button/__examples__/prevent-double-click.njk

```diff
@@ -0,0 +1,30 @@
+{% from "moduk/components/button/macro.njk" import modukButton %}
+
+<form id="testForm">
+  {{ modukButton({
+    text: "Confirm and send",
+    preventDoubleClick: true
+  }) }}
+</form>
+
+<div data-testid="submitCountDisplay"></div>
+<div data-testid="buttonCountDisplay"></div>
+
+<script>
+  var submitCount = 0
+  var buttonCount = 0
+  
+  var submitCountDisplay = document.querySelector('[data-testid="submitCountDisplay"]')
+  var buttonCountDisplay = document.querySelector('[data-testid="buttonCountDisplay"]')
+
+  document.getElementById('testForm').addEventListener('click', function () {
+    buttonCount += 1
+    buttonCountDisplay.innerText = 'Button click count '+ buttonCount;
+  })
+  
+  document.getElementById('testForm').addEventListener('submit',function (event) {
+    event.preventDefault()
+    submitCount += 1
+    submitCountDisplay.innerText = 'Form submit count '+ submitCount;
+  })
+</script>
```

Comment:
I'm unsure if this is needed but I wanted to check that the mod.js is working for preventDoubleClick attribute